### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by andreikabig** [zsviczian/obsidian-excalidraw-plugin#1599]
Ctrl + z, ctrl + y does not work.

**Comment by zsviczian** [comment]
yes - that is a know issue. There are multiple keyboard shortcuts that interfere with Obsidian shortcuts and don't work depending on Obsidain settings, device type, etc.
